src/main/java/net/dv8tion/jda/api/requests/restaction/pagination/EntitlementPaginationAction.java

```diff
@@ -85,21 +86,38 @@ public interface EntitlementPaginationAction extends PaginationAction<Entitlemen
      * @param guildId
      *        The guild id to filter by
      *
+     * @throws java.lang.IllegalArgumentException
+     *         If the provided {@code guildId} is {@code null}, empty or is not a valid snowflake
+     * @throws java.lang.NumberFormatException
+     *         If the provided {@code guildId} is negative or has a value greater than {@link Long#MAX_VALUE Long.MAX_VALUE}
```

Comment:
This is implied by not valid snowflake

Reply:
Can you also add docs for exceptions on `skuIds`

Reply:
Fair point, i also should have noticed all of these. Sorry